libsolidity/ast/ASTJsonConverter.cpp

```diff
@@ -269,7 +270,10 @@ bool ASTJsonConverter::visit(ContractDefinition const& _node)
 		make_pair("contractKind", contractKind(_node.contractKind())),
 		make_pair("abstract", _node.abstract()),
 		make_pair("baseContracts", toJson(_node.baseContracts())),
-		make_pair("contractDependencies", getContainerIds(_node.annotation().contractDependencies, true)),
+		make_pair("contractDependencies", getContainerIds(
+			_node.annotation().contractDependencies | boost::adaptors::map_keys,
```

Comment:
You could use ranges v3 here:
```suggestion
			_node.annotation().contractDependencies | views::keys,
```

This would also require these at the top:
```c++
#include <range/v3/view/map.hpp>
using namespace ranges;
```